shells/pipes-shell/source/pipe.js

```diff
@@ -44,14 +44,13 @@ export const busReady = async (bus, {manifest}) => {
 
 const configureRuntime = async ({rootPath, urlMap, storage, manifest}, bus) => {
   // configure arcs runtime environment
-  runtime = Runtime.init(rootPath, urlMap);
-  // marshal and bind context
-  const context = await requireContext(manifest);
-  runtime.bindContext(context);
+  runtime = new Runtime({rootPath, urlMap});
+  // marshal context
+  runtime.context = await requireContext(manifest);
```

Comment:
i think `context` should be readonly in Runtime. 
this is a bit of a chicken and egg problem, i remember facing it too.
parsing is async, so you'd either need a static createRuntime method, or parse it ahead of creating the Runtime, by directly calling `Manifest.parse`, not sure what's best?